python/paddle/fluid/tests/unittests/op_test.py

```diff
@@ -1190,6 +1196,10 @@ def check_output(self,
             (hasattr(self, "attrs") and "use_mkldnn" in self.attrs and \
                     self.attrs["use_mkldnn"] == True):
             self.__class__.use_mkldnn = True
+        if (hasattr(self, "use_ngraph") and self.use_ngraph == True) or \
+            (hasattr(self, "attrs") and "use_ngraph" in self.attrs and \
+                    self.attrs["use_ngraph"] == True):
+            self.__class__.use_ngraph = True
```

Comment:
Do all ngraph op tests have use_ngraph in self or self.attrs? 

Just like skipping the mkldnn op test,  it is necessary to save `self.__class__.use_ngraph` in _append_ops(). function.